ui/components/multichain/asset-picker-amount/index.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+import {
+  getSendAmount,
+  getSendAsset,
+  sendAmountIsInError,
+  updateSendAmount,
+} from '../../../ducks/send';
+import AssetPickerAmount from './asset-picker-amount';
+
+export default connect(mapStateToProps, mapDispatchToProps)(AssetPickerAmount);
```

Comment:
We've stopped using this old patterns.  Instead, with our functional components, we use `useSelector` to grab this information.